api/tests/local_providers/cinema_providers/ems/ems_stocks_test.py

```diff
@@ -53,13 +57,222 @@ def should_fill_and_create_offer_and_product_and_stock_information_for_each_movi
+        assert len(created_products) == 1
+        assert len(created_stocks) == 1
+        assert len(created_price_categories) == 1
+        assert len(created_price_categories_labels) == 1
+
+        assert created_offers[0].name == "Mon voisin Totoro"
+        assert created_offers[0].product == created_products[0]
+        assert created_offers[0].venue == venue_provider.venue
+        assert (
+            created_offers[0].description
+            == "Mei, 4 ans, et Satsuki, 10 ans, s’installent à la campagne avec leur père pour se rapprocher de l’hôpital où séjourne leur mère. Elles découvrent la nature tout autour de la maison et, surtout, l’existence d’animaux étranges et merveilleux, les Totoros, avec qui elles deviennent très amies. Un jour, alors que Satsuki et Mei attendent le retour de leur mère, elles apprennent que sa sortie de l’hôpital a été repoussée. Mei décide alors d’aller lui rendre visite seule. Satsuki et les gens du village la recherchent en vain. Désespérée, Satsuki va finalement demander de l’aide à son voisin Totoro."
+        )
+
+        assert created_offers[0].durationMinutes == 87
+        assert created_offers[0].subcategoryId == subcategories.SEANCE_CINE.id
+        assert created_offers[0].bookingEmail == "ormeaux-booking@example.com"
+        assert created_offers[0].withdrawalDetails == "Modalité de retrait"
+
+        assert created_products[0].name == "Mon voisin Totoro"
+        assert (
+            created_products[0].description
+            == "Mei, 4 ans, et Satsuki, 10 ans, s’installent à la campagne avec leur père pour se rapprocher de l’hôpital où séjourne leur mère. Elles découvrent la nature tout autour de la maison et, surtout, l’existence d’animaux étranges et merveilleux, les Totoros, avec qui elles deviennent très amies. Un jour, alors que Satsuki et Mei attendent le retour de leur mère, elles apprennent que sa sortie de l’hôpital a été repoussée. Mei décide alors d’aller lui rendre visite seule. Satsuki et les gens du village la recherchent en vain. Désespérée, Satsuki va finalement demander de l’aide à son voisin Totoro."
+        )
+        assert created_products[0].durationMinutes == 87
+        assert created_products[0].subcategoryId == subcategories.SEANCE_CINE.id
+
+        assert not created_stocks[0].quantity
+        assert created_stocks[0].price == Decimal("7.00")
+        assert created_stocks[0].dateCreated is not None
+        assert created_stocks[0].bookingLimitDatetime == datetime.datetime(2023, 7, 27, 10)
+        assert created_stocks[0].beginningDatetime == datetime.datetime(2023, 7, 27, 10)
+        assert created_stocks[0].priceCategory.price == Decimal("7.00")
+        assert created_stocks[0].priceCategory.label == "Tarif pass Culture 7€"
+        assert created_stocks[0].features == ["VF"]
+
+        # Ensuring we are keeping tracks of current version of our stocks
+        assert cinema_detail.lastVersion == mock_data["version"]
```

Comment:
```suggestion
        assert cinema_detail.lastVersion == 86400
```
Il me semble que l'on expliciter les versions attendues (et celles plus bas).